packages/components/src/components/Tabs/Tabs.stories.tsx

```diff
@@ -276,3 +276,50 @@ WithDivider.parameters = {
     },
   },
 };
+
+const Foo = () => (
+  <Box.div
+    borderColor="colorBackgroundComplete"
+    borderStyle="borderStyleSolid"
+    borderWidth="borderWidth10"
+    h="100%"
+    w="100%"
+  >
+    <h3>Foo</h3>
+    Lorem ipsum dolor sit, consectetur. Aliquam ac a. Ligula at et, sodales vel
+    purus.
+  </Box.div>
+);
+
+const Bar = () => <div>Bar</div>;
```

Comment:
```suggestion
const Bar = () => (
  <Box.div padding="space60">
    <Heading as="h3" marginBottom="space0" size="heading50">
      Bar
    </Heading>
  </Box.div>
);
```

Reply:
Why adding so much code for a story for an element that is not the focus of the story?
Isn't this distracting from the actual use case?